setup.py

```diff
@@ -6,9 +6,9 @@
 setup(
     name="atlas-consortia-commons",
     # Test PyPi version
-    # version="1.0.9",
+    # version="1.1.0",
```

Comment:
For semantic versioning this should change to `1.0.10`. Since this is on PyPi test it isn't a big deal.